index.html

```diff
@@ -394,6 +394,37 @@ <h2>
         information also discloses the location of the user of the device,
         thereby potentially compromising the user's privacy.
       </p>
+      <section class="informative">
+        <h3>
+          User consent
+        </h3>
+        <p>
+          The <cite>Geolocation API</cite> is a [=powerful feature=] that
+          requires [=express permission=] from an end-user's before any
+          location data is shared with a web application. This requirement is
+          normatively enforced by the [=check permission=] steps on which the
+          {{Geolocation/getCurrentPosition()}} and
+          {{Geolocation/watchPosition()}} methods rely.
+        </p>
+        <p>
+          A end-user will generally gives [=express permission=] through a user
+          interface, which usually present a range of permission
+          [=permission/lifetimes=] that the end-user can choose from. The
+          choice of [=permission/lifetimes=] vary across user agent, but they
```

Comment:
Nit: s/vary across user agent/could vary across user agents/

Reply:
Thanks for checking it!